crates/transaction-pool/src/pool/blob.rs

```diff
@@ -214,11 +260,47 @@ impl<T: PoolTransaction> Ord for BlobTransaction<T> {
     }
 }
 
+/// The blob step function, attempting to compute the delta given the `max_tx_fee`, and
+/// `current_fee`.
+///
+/// The `max_tx_fee` is the maximum fee that the transaction is willing to pay, this
+/// would be the priority fee for the EIP1559 component of transaction fees, and the blob fee cap
+/// for the blob component of transaction fees.
+///
+/// The `current_fee` is the current value of the fee, this would be the base fee for the EIP1559
+/// component, and the blob fee (computed from the current head) for the blob component.
+///
+/// This is supposed to get the number of fee jumps required to get from the current fee to the fee
+/// cap, or where the transaction would not be executable any more.
+fn fee_delta(max_tx_fee: u128, current_fee: u128) -> f64 {
+    // jumps = log1.125(txfee) - log1.125(basefee)
+    // TODO: should we do this without f64?
+    let jumps = (max_tx_fee as f64).log(1.125) - (current_fee as f64).log(1.125);
+    // delta = sign(jumps) * log(abs(jumps))
+    jumps.signum() * jumps.abs().log2()
+}
+
+/// Returns the priority for the transaction, based on the "delta" blob fee and priority fee.
+fn blob_tx_priority(
+    blob_fee_cap: u128,
+    blob_fee: u128,
+    max_priority_fee: u128,
+    base_fee: u128,
+) -> f64 {
+    let delta_blob_fee = fee_delta(blob_fee_cap, blob_fee);
+    let delta_priority_fee = fee_delta(max_priority_fee, base_fee);
+
+    // priority = min(delta-basefee, delta-blobfee, 0)
+    delta_blob_fee.min(delta_priority_fee).min(0.0)
+}
+
 #[derive(Debug, Clone)]
 struct BlobOrd {
     /// Identifier that tags when transaction was submitted in the pool.
     pub(crate) submission_id: u64,
```

Comment:
looks like we no longer need this because we only have a single dimension with f64

Reply:
right, makes sense, removed the field from the subpool altogether